ratox.c

```diff
@@ -1188,6 +1188,16 @@ toxinit(void)
 
 	free((void *)toxopt.savedata_data);
 
+	#if TOX_VERSION_PATCH >= 1
```

Comment:
tox_version_patch will be 0 at v0.1.0, but the API will still have changed.

This might not be a bad thing, a minor version change could be cause to throw an error, but this isn't very forward compatible.